dlib/cmake_utils/find_blas.cmake

```diff
@@ -50,7 +58,19 @@ if (UNIX OR MINGW)
 
       # We need cblas since MATLAB doesn't provide cblas symbols.
       add_subdirectory(external/cblas)
-      set(blas_libraries  cblas  )
+      if("${CMAKE_SYSTEM}" MATCHES "Linux")
+        if("${LSB_RELEASE_ID_SHORT}" STREQUAL "Arch Linux" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "Arch" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "ArcoLinux" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "EndeavourOS" OR 
+	    "${LSB_RELEASE_ID_SHORT}" STREQUAL "ManjaroLinux")
+	  	 set(blas_libraries "-lcblas;-lblas")
+	  else()
+	    set(blas_libraries  cblas)
+	  endif()
+      else()
+        set(blas_libraries  cblas)
+      endif()
```

Comment:
Are you building matlab extensions?  Like did you test this code path?

Reply:
I didn't build Matlab Extensions, I just use same working code to the same condition.